scipy/sparse/linalg/_isolve/_gcrotmk.py

```diff
@@ -221,10 +221,10 @@ def gcrotmk(A, b, x0=None, *, rtol=1e-5, atol=0., maxiter=1000, M=None, callback
         According to [2]_, good values are around m.
         Default: m
     CU : list of tuples, optional
-        List of tuples ``(c, u)`` which contain the columns of the matrices
+        List of tuples `(c, u)` which contain the columns of the matrices
```

Comment:
```suggestion
        List of tuples ``(c, u)`` which contain the columns of the matrices
```

`(c, u)` isn't a parameter name (rather a representation of an argument), so we want double backticks here too